scripts/update-github-actions.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# This script calls the dependabot CLI (https://github.com/dependabot/cli)
+# to determine updates to GitHub Action dependencies in the local repository.
+# It then also applies the updates and outputs the results to standard output.
+
+set -euo pipefail
+
+REPO_ROOT=$1
+
+echo -e "<details><summary>GitHub Action updates</summary>\n\n"
+
+# Each dependabot update call tries to update all dependencies,
+# but the resulting files are output individually for each (with the intention of creating a PR for each).
+# We want to have all changes together though, so we just repeatedly take the first one
+# until there's none anymore (-e jq flag)
+while
+  # Unused argument would be the remote GitHub repo, which is not used if we pass --local
+  create_pull_request=$(LOCAL_GITHUB_ACCESS_TOKEN=$(gh auth token) \
+    dependabot update github_actions this-argument-is-unused --local "$REPO_ROOT" \
+    | jq -ecs 'map(select(.type == "create_pull_request")) | .[0].data')
```

Comment:
Wow, this is one heck of a pipeline.

I might:

1. Extract `gh auth token` as it won't change in the loop
2. Use the longer form version of `jq` arguments (`--exit-status`, `--compact-output`, `--slurp`)

It's serviceable as is, though.

Reply:
Done :)